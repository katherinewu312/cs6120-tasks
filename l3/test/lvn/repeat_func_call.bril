@main {
  v: int = const 4;
  x: int = const 2;
  x: int = call @add v x;
  x: int = call @add x w;
  y: int = const 2;
  print x;
}

@add(a: int, b:int): int{
    ab: int = add a b;
    ret ab;
}