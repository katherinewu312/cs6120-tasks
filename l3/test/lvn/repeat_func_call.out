@main {
  v: int = const 4;
  x_1: int = const 2;
  x_2: int = call @add v x_1;
  x: int = call @add x_2 w;
  y: int = id x_1;
  print x;
}
@add(a: int, b: int): int {
  ab: int = add a b;
  ret ab;
}
